presto-native-execution/presto_cpp/main/operators/BroadcastWrite.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/operators/BroadcastWrite.h"
+#include <boost/lexical_cast.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+using namespace facebook::velox::exec;
+using namespace facebook::velox;
+
+namespace facebook::presto::operators {
+namespace {
+velox::core::PlanNodeId deserializePlanNodeId(const folly::dynamic& obj) {
+  return obj["id"].asString();
+}
+
+class BroadcastWriteOperator : public Operator {
+ public:
+  BroadcastWriteOperator(
+      int32_t operatorId,
+      DriverCtx* FOLLY_NONNULL ctx,
+      const std::shared_ptr<const BroadcastWriteNode>& planNode)
+      : Operator(
+            ctx,
+            planNode->outputType(),
+            operatorId,
+            planNode->id(),
+            "BroadcastWrite") {
+    const auto& broadcastBasePath = planNode->broadcastWriteBasePath();
+    // fmt::format("{}/{}", planNode->broadcastWriteBasePath(), makeUuid());
+    auto fileBroadcast = FileBroadcast(broadcastBasePath);
```

Comment:
fileBroadcast and fileBroadcast_ variable names are very similar, but they represent different things; consider renaming